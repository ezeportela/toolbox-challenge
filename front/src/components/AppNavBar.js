import React from "react";
import Container from "react-bootstrap/Container";
import Navbar from "react-bootstrap/Navbar";

const AppNavBar = () => {
  return (
    <Navbar expand="lg" className="bg-danger" variant="dark" fixed="top">
      <Container>
        <Navbar.Brand href="#home">React Test App</Navbar.Brand>
      </Container>
    </Navbar>
  );
};

export default AppNavBar;
