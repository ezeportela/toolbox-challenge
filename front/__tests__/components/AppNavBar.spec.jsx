import React from "react";
import AppNavBar from "../../src/components/AppNavBar";
import renderer from "react-test-renderer";

describe("AppNavBar (Snapshot)", () => {
  it("Should render component", () => {
    const component = renderer.create(<AppNavBar />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
