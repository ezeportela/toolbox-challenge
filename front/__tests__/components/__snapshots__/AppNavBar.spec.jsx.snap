// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNavBar (Snapshot) Should render component 1`] = `
<nav
  className="bg-danger navbar navbar-expand-lg navbar-dark fixed-top"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="#home"
    >
      React Test App
    </a>
  </div>
</nav>
`;
